ckan_cloud_operator/logs.py

```diff
@@ -74,7 +74,7 @@ def log_subprocess_output(stdout, stderr):
 def subprocess_run(command, input=None):
     completed = subprocess.run(
         command, input=input, 
-        shell=True, check=True, capture_output=True
+        shell=True, check=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE
```

Comment:
why?

Reply:
Compatibility with Python 3.6 (`capture_output` introduced in 3.7 and does [the same](https://github.com/python/cpython/blob/3.7/Lib/subprocess.py#L481))